main.tf

```diff
@@ -62,14 +62,19 @@ resource "helm_release" "cloud_monitoring_agent" {
     value = local.cluster_name
   }
   set {
-    name  = "image.version"
+    name  = "global.imageRegistry"
     type  = "string"
-    value = var.image_tag_digest
+    value = var.image_registry
   }
   set {
-    name  = "image.registry"
+    name  = "image.repository"
```

Comment:
I don't see `image.repository` listed in the docs? I don't think its needed if you fix the value of `global.imageRegistry` as above comment?

Reply:
Ignore - I think I'm wrong